packages/scan/src/DefaultScans.spec.ts

```diff
@@ -110,7 +110,7 @@ describe('HttpScans', () => {
 
       await scans.createScan({
         name: 'test',
-        tests: [TestType.DOM_XSS],
+        tests: [TestType.STORED_XSS],
```

Comment:
`dom_xss` is superseded by `xss` rather than `stored_xss`
```suggestion
        tests: [TestType.XSS],
```